Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+ADD . /repo
+WORKDIR /repo
+RUN pip3 install .
+RUN rm -rf /repo
+
+CMD [ "operator-courier", "--help"]
```

Comment:
Please add a newline at the end of this file.

Reply:
done